### PR TITLE
chore: remove instrumentation for email performance monitoring and SES migration

### PR DESCRIPTION
**Pull request by justynoh**
## Problem

We created additional instrumentation over our mailing system in #5019 and #5369 and #5442. This PR tears them down, since they are not needed anymore.

Closes https://github.com/opengovsg/formsg-private/issues/130 and closes https://github.com/opengovsg/formsg-private/issues/132.

Cleaning up old SES config will also close https://github.com/opengovsg/formsg-private/issues/91.

## Solution

Delete code that references datadog traces and SES in the US.

**Breaking Changes** 
- No - this PR is backwards compatible  

## Tests

- [x] OTP emails can be sent to admins to log in.
- [x] Email responses can be sent to admins after submitting an email mode form.
- [x] OTP emails can be sent to respondents when verifying an email in a public form.
- [x] Confirmation emails can be sent to respondents when submitting and email mode form.

## Deploy Notes

**New environment variables**:

for `x` = `HOST`, `PORT`, `USER`, `PASS`:

- `SES_{x}` : Should be the value of the old variable `SES_{x}_SG`
- `SES_{x}_US` and `SES_{x}_SG`: Should be deleted

### CODE REVIEW

**Comment by justynoh** [comment]
(This PR must be released before we tear down the infra for SES US)

**Review by timotheeg — COMMENTED** [review]
Looking good!

Unless the tracing of the email sending is costing a lot (I assume no right now), let's keep them in for now (traces for email sending + pdf generation), it gives us visibility in processes we are otherwise blind to.

File: src/app/services/mail/mail.service.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { tracer } from 'dd-trace'
+import tracer from 'dd-trace'
```
Comment (timotheeg) [inline src/app/services/mail/mail.service.ts:1]: is this change intended?

Comment (justynoh) [reply src/app/services/mail/mail.service.ts:1]: oops no, but it shouldn't make a difference! it was probably introduced when I reinstated the tracer spans. But it still works, tested on staging + the default import was already used in `mail.utils.ts` before

**Review by timotheeg — APPROVED** [review]
LGTM
